dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/OidcClientAuthenticator.java

```diff
@@ -164,7 +164,7 @@ private ProviderAuthenticationResult invokeUserResolverSPI(ProviderAuthenticatio
      * @param result
      * @return
      */
-    ProviderAuthenticationResult fixSubject(ProviderAuthenticationResult result) {
+    public ProviderAuthenticationResult fixSubject(ProviderAuthenticationResult result) {
```

Comment:
This method should be static.  Also not sure why this class couldn't move as well.